### ISSUE TITLE
Run tests on Netty for WebFlux

### ISSUE THREAD

**Issue by malafeev** [opentracing-contrib/java-spring-web#112]
WebFlux doesn't require jetty or tomcat. 
It uses netty.
Is it possible to get rid of jetty from WebFlux integration tests?

**Comment by geoand** [comment]
Is this causing problems for you for some reason?

**Comment by malafeev** [comment]
yes, it looks like TracingSubscriber.onComplete() is never called, therefore span is not finished:
https://github.com/opentracing-contrib/java-spring-web/blob/master/opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/webfilter/TracingSubscriber.java#L77

**Comment by geoand** [comment]
So I might be missing something here... Is this tied to another issue perhaps?

**Comment by malafeev** [comment]
If I use in my app tomcat then I see that `TracingSubscriber.onComplete()` is called.
But if I don't use tomcat and rely on netty which comes with spring-webflux I don't see calling TracingSubscriber.onComplete().
That's why I'm curious why integration test for WebFlux is based on jetty.
I use the latest version of spring.

**Comment by geoand** [comment]
@csabakos can you shed some light on this perhaps since you were the one that added those tests?

Thanks

**Comment by csabakos** [comment]
@geoand WebFlux should work equally over any runtime (Tomcat, Jetty, Netty, etc.), so there was no reason to switch away from the runtime that was already used in other tests.

If the observed behavior is different on Netty, then I think it would make sense to add Netty-based integration tests to make sure that all cases are covered.

**Comment by geoand** [comment]
So you just added like that for convenience, thanks that's good to know.

@malafeev Would you like to take a stab at it? If you are unavailable, I can take a look but it will probably be a while...

**Comment by malafeev** [comment]
@geoand it would be great if you take a look even in a while. Currently I don't have enough time.

**Comment by geoand** [comment]
Same here :).

I'll take a look as soon as I can

**Comment by geoand** [comment]
I want to upgrade to Opentracing 0.33 first. Then I'll look into this
